src/hotspot/share/prims/methodHandles.cpp

```diff
@@ -436,7 +436,6 @@ Symbol* MethodHandles::signature_polymorphic_intrinsic_name(vmIntrinsics::ID iid
   case vmIntrinsics::_linkToNative:     return vmSymbols::linkToNative_name();
   default:
     fatal("unexpected intrinsic id: %d %s", vmIntrinsics::as_int(iid), vmIntrinsics::name_at(iid));
-    return 0;
```

Comment:
Do we no longer need these returns after `fatal` to keep compilers happy?

Reply:
Now that we have, and are using, `[[noreturn]]` on all platforms, we no longer need that dead code.

Reply:
I'll admit, I do prefer having a return to end all possible control flows in a non void method, but oh well

Reply:
I would rather it not look like it can return null (or some other manufactured "default") when it actually can't.